src/integrationTest/resources/db/testmigration/V1_6__excption_table.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE dataload_exception_records(
+                                           id SERIAL NOT NULL,
+                                           file_name varchar(256),
+                                           table_Name varchar(64),
+                                           scheduler_Start_Time timestamp NOT NULL,
+                                           key varchar(256),
+                                           field_in_error varchar(256),
+                                           error_description varchar(512),
+                                           updated_timestamp timestamp NOT NULL,
+                                           CONSTRAINT dataload_exception_records_pk PRIMARY KEY (ID)
```

Comment:
table definition does not match with  the one provided in confluence. Change it accordingly

Reply:
will change after merge as I need to accomodate scheduler name